Modules/MessageUtils/PaginatedEmbed.js

```diff
@@ -1,75 +1,76 @@
 const { Colors, PageEmojis } = require("../../Internals/Constants");
 
 class PaginatedEmbed {
-	constructor (msg, descriptions = [], embed = null, images = [], fields = [], thumbnails = []) {
-		this.client = msg.client;
-		this.originalMsg = msg;
-
+	/**
+	 * After creating a PaginatedEmbed call `#init()` to set it up and start listenig for reactions.
+	 *
+	 * @param originalMsg 	The original message that created this paginated embed.
+	 * 						May be a custom object, the only required fields are `channel` and `author.id`
+	 * @param embedTemplate A slightly edited embed object that serves as the base template for all pages,
+	 * 						with strings being formatted via templates
+	 * @param pageData		All the data used for the different pages of the embed pages,
+	 * 						with the fields being arrays with values (or null) for every page
+	 */
+	constructor (originalMsg, embedTemplate, pageData) {
```

Comment:
`pageData` can be turned into `{ authors = [], titles = [], colors = [] } = {}` and so on